Bottom: 0a12913e0c140f238c152fc43de75713d9505d15
Top:    5ee88635807a574a198814a07fec8dd2c8ad46b2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 17:07:47 +0100

Win32: Disable the SpaceNavigator code


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 158d646..2cefb48 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -591,8 +591,8 @@ ghid_port_window_enter_cb (GtkWidget * widget,
       /* Make sure drawing area has keyboard focus when we are in it.
        */
       gtk_widget_grab_focus (out->drawing_area);
-      if (ghidgui->snavi)
-        snavi_set_led (ghidgui->snavi, TRUE);
+      //if (ghidgui->snavi)
+      //  snavi_set_led (ghidgui->snavi, TRUE);
     }
   ghidgui->in_popup = FALSE;
 
@@ -695,8 +695,8 @@ ghid_port_window_leave_cb (GtkWidget * widget,
 	    }
 	  g_idle_add (ghid_pan_idle_cb, NULL);
 	}
-      else if (ghidgui->snavi)
-        snavi_set_led (ghidgui->snavi, FALSE);
+      //else if (ghidgui->snavi)
+      //  snavi_set_led (ghidgui->snavi, FALSE);
     }
 
   out->has_entered = FALSE;
diff --git a/src/hid/gtk/snavi.c b/src/hid/gtk/snavi.c
index 2f58f02..d5c5850 100644
--- a/src/hid/gtk/snavi.c
+++ b/src/hid/gtk/snavi.c
@@ -12,6 +12,7 @@
  *
  */
 
+#if 0
 #include <stdio.h>
 #include <errno.h>
 #include <stdlib.h>
@@ -21,9 +22,11 @@
 
 #include <linux/types.h>
 #include <linux/input.h>
+#endif
 
 #include <glib.h>
 
+#if 0
 #define test_bit(bit, array)  (array [bit / 8] & (1 << (bit % 8)))
 
 static void (*update_pan_cb)(int, int, int, gpointer);
@@ -150,6 +153,7 @@ gboolean snavi_event (GIOChannel   *source,
 
   return TRUE;
 }
+#endif
 
 
 GIOChannel *
@@ -159,6 +163,9 @@ setup_snavi (void (*update_pan)(int, int, int, gpointer),
              void (*button)(int, int, gpointer),
              gpointer data)
 {
+  return NULL;
+
+#if 0
   GIOChannel *snavi;
 
   update_pan_cb = update_pan;
@@ -186,4 +193,5 @@ setup_snavi (void (*update_pan)(int, int, int, gpointer),
     }
 
   return snavi;
+#endif
 }
